src/Service/Lambda/README.md

```diff
@@ -10,3 +10,22 @@ An API client for Lambda.
 ```cli
 composer require async-aws/lambda
 ```
+
+## Usage
+
+See [docs/authentication.md](https://github.com/async-aws/aws/blob/master/docs/authentication.md) to see how to authenticate against AWS server.
+
+```php
+$client = new LambdaClient([
+    'region' => 'us-east-1',
+]);
+
+$result = $client->invoke([
+  'FunctionName' => 'app-dev-hello_world', // Find it here: https://console.aws.amazon.com/lambda/home?region=us-east-1#/functions
+  'Payload' => '{"name": "async-aws/lambda"}',
+]);
+
+$result->resolve();
```

Comment:
this is not needed.

`resolve` allows you to manually wait for the response when you do nothing with `$result`. This is for special edge case.